manim/utils/manim_banner.py

```diff
@@ -0,0 +1,100 @@
+"""Manim Banner utilities."""
+
+__all__ = ["ManimBanner"]
+
+from ..constants import LEFT, UP, RIGHT, DOWN, ORIGIN
+from ..animation.composition import AnimationGroup
+from ..animation.fading import FadeIn
+from ..animation.transform import ApplyMethod
+from ..mobject.geometry import Circle, Square, Triangle
+from ..mobject.svg.tex_mobject import Tex, MathTex
+from ..mobject.types.vectorized_mobject import VGroup
+from ..utils.tex_templates import TexFontTemplates
+
+
+class ManimBanner(VGroup):
+    r"""Convenience class representing Manim's banner.
+
+    Can be animated using the costum methods.
+
+    Examples
+    --------
+
+    .. manim:: BannerExample
+
+        from manim.utils.manim_banner import ManimBanner
+
+        class BannerExample(Scene):
+            def construct(self):
+                banner = ManimBanner().scale(0.25).to_corner(DR)
+                self.add(banner)
+                self.wait()
+                self.play(banner.expand())
+                self.wait()
+
+    """
+
+    def __init__(self, dark_theme=True):
+        VGroup.__init__(self)
+
+        logo_green = "#81b29a"
+        logo_blue = "#454866"
+        logo_red = "#e07a5f"
+        m_height_over_anim_height = 0.75748
+
+        self.font_color = "#ece6e2" if dark_theme else "#343434"
+        self.scale_factor = 1
+
+        self.M = MathTex(r"\mathbb{M}").scale(7).set_color(self.font_color)
+        self.M.shift(2.25 * LEFT + 1.5 * UP)
+
+        self.circle = Circle(color=logo_green, fill_opacity=1).shift(LEFT)
+        self.square = Square(color=logo_blue, fill_opacity=1).shift(UP)
+        self.triangle = Triangle(color=logo_red, fill_opacity=1).shift(RIGHT)
+        self.add(self.triangle, self.square, self.circle, self.M)
+        self.move_to(ORIGIN)
+
+        anim = VGroup()
+        for i, ch in enumerate("anim"):
+            tex = Tex(
+                "\\textbf{" + ch + "}",
+                tex_template=TexFontTemplates.gnu_freeserif_freesans,
```

Comment:
I would guess that this line is making the docs fail, I remember there was something similar two month ago.
Maybe we can try to init the tex_template like this: 
https://docs.manim.community/en/latest/reference/manim.mobject.svg.tex_mobject.html#addpackagelatex

Reply:
We just need to change the output format for this template from pdf to xdv, I had to do this locally as well.